File: beecrowd/1035 - Teste de Seleção 1.rs
use std::io;

fn main() {

    let mut entrada: String = String::new();

    io::stdin()
        .read_line(&mut entrada)
        .expect("Falha ao ler uma linha");

    let valores: Vec<i32> = entrada.trim()
                                .split(" ")
                                .map(|x: &str| x.trim().parse::<i32>().expect("Falha ao interpretar como inteiro"))
                                .collect();

    let a = valores[0];
    let b = valores[1];
    let c = valores[2];
    let d = valores[3];

    if b > c && d > a && c + d > a + b && c > 0 && d > 0 && a % 2 == 0 {
        println!("Valores aceitos");
    } else {
        println!("Valores nao aceitos");
    }
}

File: beecrowd/1035 - Teste de Seleção 1.kt
import java.util.*

fun canAcceptValues(a: Int, b: Int, c: Int, d: Int): Boolean = b > c && d > a && c + d > a + b && c > 0 && d > 0 && a % 2 == 0

fun main(args: Array<String>) {

    val line = readLine()!!

    val elements = line.split(' ')

    val a = elements[0].toInt()
    val b = elements[1].toInt()
    val c = elements[2].toInt()
    val d = elements[3].toInt()

    if (canAcceptValues(a, b, c, d)) {
        println("Valores aceitos")
    } else {
        println("Valores nao aceitos")
    }
}